module/spl/spl-condvar.c

```diff
@@ -311,9 +311,8 @@ cv_timedwait_hires_common(kcondvar_t *cvp, kmutex_t *mp, hrtime_t tim, hrtime_t
 		tim = (tim / res) * res;
 	}
 
-	ASSERT(!(flag & CALLOUT_FLAG_ABSOLUTE));
-	/* get abs expire time */
-	tim += gethrtime();
+	if (!(flag & CALLOUT_FLAG_ABSOLUTE))
+		tim += gethrtime();
```

Comment:
I'm not sure how I missed this when I looked over 39cd90e but yes we definitely need to restore this.